pyanaconda/kickstart.py

```diff
@@ -682,6 +679,8 @@ def ksshowwarning(message, category, filename, lineno, file=None, line=None):
                 if not report.is_valid():
                     message = "\n\n".join(map(str, report.error_messages))
                     raise KickstartError(message)
+                if strict_mode:
```

Comment:
I think that you can always add warnings from DBus modules to `kswarnings`. I suggest to do that before raising  `KickstartError`, because we print the collected warnings in the exception handler in the strict mode.

Reply:
Thank you - done.